examples/library/src/Server.hs

```diff
@@ -42,12 +40,36 @@ main = do
   runStderrLoggingT $
     withSqliteConn @(LoggingT IO) ":memory:" $ \conn -> do
       runDb conn $ runMigration migrateAll
+      -- Insert demo data
+      insertSeedData conn
+      liftIO $ putStrLn "starting GraphQL server on port 8000"
       liftIO $ run 8000 $ hm $
         graphQLApp (prometheus p $ libraryServer conn)
                    (Proxy @('Just "Query"))
                    (Proxy @('Just "Mutation"))
                    (Proxy @('Just "Subscription"))
 
+{- | Inserts demo data to make this example valueable for testing with different clients
+Returns Nothing in case of any failure, including attempts to insert non-unique values
+-}
+insertSeedData :: SqlBackend -> LoggingT IO (Maybe ())
+insertSeedData conn = sequence_ <$> sequence
+  [ insertAuthorAndBooks conn (Author "Robert Louis Stevenson")
+      [Book "Treasure Island", Book "Strange Case of Dr Jekyll and Mr Hyde"]
+  , insertAuthorAndBooks conn (Author "Immanuel Kant")
+      [Book "Critique of Pure Reason"]
+  , insertAuthorAndBooks conn (Author "Michael Ende")
+      [Book "The Neverending Story", Book "Momo"]
+  ]
+
+{- | Inserts Author and Books
+Returns Nothing in case of any failure, including attempts to insert non-unique values
+-}
+insertAuthorAndBooks :: SqlBackend -> Author -> [Key Author -> Book] -> LoggingT IO (Maybe ())
+insertAuthorAndBooks conn author books =
+  runDb conn $ (>>= sequence_) <$> do
+    authorResult <- insertUnique author
+    traverse (\authorId -> traverse (insertUnique . ($ authorId)) books) authorResult
```

Comment:
Similar here, we could write a much easier version by eta-expanding.

```haskell
   insertUnique . ($ authorId)
= \kBook -> insertUnique (kBook $ authorId)
= \kBook -> insertUnique (kBook authorId)
```

Reply:
Changed it, thank you.